src/types/pathways-model.d.ts

```diff
@@ -32,6 +32,7 @@ declare module 'pathways-model' {
     label: string;
     transitions: Transition[];
     nodeTypeIsUndefined?: boolean;
+    type: 'action' | 'branch' | 'reference' | 'other';
```

Comment:
What falls under the `other` category? I noticed null nodes still use `nodeTypeIsUndefined`. Perhaps we can consolidate these two together
```suggestion
    type: 'action' | 'branch' | 'reference' | 'null' | 'start';
```

Reply:
Looks like 'other' just means start nodes, so I agree with making that explicit. Did you mean `'null'` or just `null` here?